Gems/Multiplayer/Code/Source/Pipeline/NetworkPrefabProcessor.cpp

```diff
@@ -173,6 +196,7 @@ namespace Multiplayer
         AZ::Data::AssetId networkSpawnableAssetId(context.GetSourceUuid(), AzFramework::SpawnableAssetHandler::BuildSubId(uniqueName));
```

Comment:
Because of the way the code is written (lot of output parameters), this is currently hard to refactor. So, this is something that you can look into as a follow up. But everything here technically can be skipped if there are no network entities to begin with.